core/fixnum/plus_spec.rb

```diff
@@ -17,4 +17,12 @@
     lambda { 13 + "10"    }.should raise_error(TypeError)
     lambda { 13 + :symbol }.should raise_error(TypeError)
   end
+  it "overflows to Bignum when the result does not fit in Fixnum" do
```

Comment:
Minor stuff, but you probably want to ad an empty line before the `it` and indent the subsequent lines with 2 spaces :wink: